lib/jsdom/browser/xhr/xmlhttprequest.js

```diff
@@ -0,0 +1,891 @@
+"use strict";
+
+var HTTP_STATUS_CODES = require("http").STATUS_CODES;
+var deasync;
+try {
+  deasync = require("deasync");
```

Comment:
Thank you! I think it's worth it. Sync XHR is an evil feature anyway and it's OK if it's slow.